db-changes/datasets/01062021_dataset_acquisition_import_date.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE dataset_acquisition ADD COLUMN creation_date date;
```

Comment:
We should totally deploy on qualif everytime, because a new column is missing here for mr_protocol.magnetic_field_strength.